_destinations/mysql/guides/mysql-setup.md

```diff
@@ -0,0 +1,152 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/destination-templates/destination-setup/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+# Page formatting & Controls #
+# -------------------------- #
+
+title: Connecting a MySQL Destination to Stitch
+permalink: /destinations/mysql/connecting-mysql-to-stitch
+
+keywords: mysql, mysql data warehouse, mysql data warehouse, mysql etl, etl to mysql, mysql destination
+summary: "Connect a MySQL destination to your Stitch account."
+
+content-type: "destination-setup"
+key: "mysql-destination-setup"
+order: 1
+
+toc: true
+layout: tutorial
+use-tutorial-sidebar: false
+
+
+# -------------------------- #
+#     Destination Details    #
+# -------------------------- #
+
+type: "mysql"
+display_name: "MySQL"
+name: "mysql"
+
+port: 3306
+
+hosting-type: "generic" # amazon, generic, microsoft, etc.
+
+api-type: "mysql"
+
+this-version: "1"
+
+ssh: true
+
+
+# -------------------------- #
+#        Introduction        #
+# -------------------------- #
+
+intro: |
```

Comment:
Double-check to see if having this section creates a blank space in the preview. If it does, you can delete the whole Introduction section from this file.

Reply:
I removed it, thanks!